src/cli2/merge.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import fs from "fs-extra";
+import stringify from "json-stable-stringify";
+import {join as pathJoin} from "path";
+
+import {LoggingTaskReporter} from "../util/taskReporter";
+import type {Command} from "./command";
+import {makePluginDir, loadInstanceConfig} from "./common";
+import {
+  toJSON as weightedGraphToJSON,
+  fromJSON as weightedGraphFromJSON,
+  type WeightedGraph,
+  merge,
+} from "../core/weightedGraph";
+
+function die(std, message) {
+  std.err("fatal: " + message);
+  return 1;
+}
+
+const mergeCommand: Command = async (args, std) => {
+  if (args.length !== 0) {
+    die(std, "usage: sourcecred merge");
+  }
+  const taskReporter = new LoggingTaskReporter();
+  taskReporter.start("merge");
+  const baseDir = process.cwd();
+  const config = await loadInstanceConfig(baseDir);
+  const graphOutputPrefix = ["output", "graphs"];
+
+  async function loadGraph(pluginName): Promise<WeightedGraph> {
+    const outputDir = makePluginDir(baseDir, graphOutputPrefix, pluginName);
```

Comment:
Do we actually want to make this directory? It doesn’t seem semantically
right here—we’re expecting a file in that directory to exist, anyway.
Maybe add `getPluginDir` parallel to `makePluginDir` (in `common`) that
just returns the pathname?

Reply:
Discussed offline, you'll make this in a separate change.